### PR TITLE
Updates contributing with Gerrit instructions

### PR DESCRIPTION
**Pull request by joehakimrahme**
Testing to see how the workflow goes

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -69,6 +69,50 @@ more info.
...
+
+4. Commit the staged files using the ``--amend`` flag. This will
+   overwrite your previous commit instead of creating a new one. This
+   will lead your patch to be confined to a single commit. It will
+   make peer review easier as well as produce cleaner, more organized
+   logs. Make sure to update your commit message if needs be.
+
+::
+
+   git commit --amed
```
Comment (stephenfin) [inline CONTRIBUTING.rst]: `--amend`

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -69,6 +69,50 @@ more info.
...
+
+
+---------------------------------------
+Submitting a new revision of your patch
+---------------------------------------
+
+After review, you may be asked to make changes to your code. Here are
+the steps you should take:
+
+1. Checkout the local branch that holds your patch.
```
Comment (stephenfin) [inline CONTRIBUTING.rst]: This is not how you should work with Gerrit. Rather, you should push everything and delete the local branch. Once you have comments to address, you download the version stored in Gerrit. This means Gerrit remains the sole source of truth.

    git review -fy  # to push change
    # wait for comments
    git review -d CHANGE_NUM  # to pull in whatever's in Gerrit

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -69,6 +69,50 @@ more info.
...
+
+2. Make modifications.
+
+3. Stage the files you just modified.
+
+::
+
+   git add files
```
Comment (stephenfin) [inline CONTRIBUTING.rst]: `git add [<FILE> ...]` ?

File: CONTRIBUTING.rst [hunk 4]
```diff
@@ -69,6 +69,50 @@ more info.
...
+   git commit --amed
+
+5. Now push the commits to the upstream branch. Since you've ammended
+   your commit, you've rewritten history and git will refuse to push
+   the changes by default. You can force it to accept your changes
+   with the ``--force`` flag.
+
+::
+
+   git push --force
```
Comment (stephenfin) [inline CONTRIBUTING.rst]: Um, can't you use `git review -fy`?

**Comment by joehakimrahme** [comment]
Hey @stephenfin, closing these very old PRs, since all contribution has to be on gerrithub now.

I didn't know of `git review -fy`. It would be great if you have time to submit a patch to modify the file with your changes.

**Pull request closed without merging**